### ISSUE TITLE
[Bug]: Upload multiple PDFs and only shows one of them in the viewer

### ISSUE THREAD

**Issue by DevShaded** [joaopaulolndev/filament-pdf-viewer#10]
### What happened?

I currently have a `FileUpload` form where a user can upload a PDF file. When I upload the first PDF file, the PDF shows in the preview. When I upload a second PDF and then save, I don't see the second PDF in the view.

### How to reproduce the bug

This is the code I added:

```php
FileUpload::make('document_management')
    ->panelLayout('grid')
    ->previewable()
    ->multiple(),

PdfViewerField::make('document_management')
    ->label('View the PDF')
    ->minHeight('40svh'),
  ```

Here is an example:

https://github.com/user-attachments/assets/6c169bcb-3a6f-4926-9a6a-75ae571df3ed

### Package Version

v1.0.5

### PHP Version

v8.3.11

### Laravel Version

v11.23.5

### Which operating systems does with happen with?

macOS

**Comment by joaopaulolndev** [comment]
This is not a bug, the plugin is now prepared just for a single PDF per field

**Comment by DevShaded** [comment]
Can I make a pull request to support multiple PDFs?

**Comment by joaopaulolndev** [comment]
@DevShaded yeah, go ahead. Just pay attention to work default as now for not break other projects.

**Comment by DevShaded** [comment]
Got it!

**Comment by jeffersongoncalves** [comment]
Hi @DevShaded, thanks for the suggestion!

As mentioned by @joaopaulolndev, the plugin is designed to display a single PDF per field. This is the intended behavior.

Since no PR was submitted for the multiple PDFs feature, we're closing this issue for now. If anyone would like to contribute this enhancement, feel free to open a new PR — just make sure to keep single PDF as the default behavior to avoid breaking existing projects.

Thanks!
